application-development/helloworld-springboot-auth/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,5 @@
+eclipse.preferences.version=1
```

Comment:
remove this file?

I don't think we should be committing any IDE configuration files.
thanks